components/omega/src/base/IO.h

```diff
@@ -68,7 +68,7 @@ enum FileFmt {
    FmtHDF5     = PIO_IOTYPE_HDF5,     ///< native HDF5 format
    FmtADIOS    = PIO_IOTYPE_ADIOS,    ///< ADIOS format
    FmtUnknown  = -1,                  ///< Unknown or undefined
-   FmtDefault  = PIO_IOTYPE_NETCDF4C, ///< NetCDF4 is default
+   FmtDefault  = PIO_IOTYPE_PNETCDF,  ///< work around HDF5 in SYCL runs
```

Comment:
Does this change, setting the default I/O type to PNETCDF, work on all machines, including Frontier, Perlmutter, and Chrysalis?